panther_hardware_interfaces/src/panther_system.cpp

```diff
@@ -476,16 +476,17 @@ void PantherSystem::ConfigureEStop()
   RCLCPP_INFO(logger_, "Successfully configured E-Stop");
 }
 
-void PantherSystem::UpdateMotorsStates()
+void PantherSystem::UpdateMotorsState()
 {
   try {
-    motors_controller_->UpdateMotorsStates();
+    motors_controller_->UpdateMotorsState();
     UpdateHwStates();
-    UpdateMotorsStatesDataTimedOut();
+    UpdateMotorsStateDataTimedOut();
   } catch (const std::runtime_error & e) {
     roboteq_error_filter_->UpdateError(ErrorsFilterIds::READ_PDO_MOTOR_STATES, true);
-    RCLCPP_WARN_STREAM_THROTTLE(
-      logger_, steady_clock_, 1000,
+
+    RCLCPP_ERROR_STREAM_THROTTLE(
+      logger_, steady_clock_, 10000,
```

Comment:
Don't you  think that 10s is too much?

Reply:
Decreased those two occurrences to 5s  👍🏼